src/Http/Http.Extensions/test/ProblemDetailsDefaultWriterTest.cs

```diff
@@ -12,6 +12,7 @@
 using System.Text.Json.Serialization;
 using Microsoft.CodeAnalysis;
 using JsonOptions = Microsoft.AspNetCore.Http.Json.JsonOptions;
+using System.Diagnostics;
```

Comment:
Please keep the `using`s sorted.

Reply:
Good catch, @gfoidl! It looks like the using in this file were previously unsorted anyways (likely because it is a test file).

@tcortega If you're able to fix the using sorting on this file overall that would be great! Otherwise, I can edit and merge.

Reply:
Resolved, also removed some unnecessary usings @gfoidl, @captainsafia